src/main/kotlin/blackjack/business/Players.kt

```diff
@@ -0,0 +1,19 @@
+package blackjack.business
+
+class Players private constructor(players: List<Player>) {
+
+    private val _players: List<Player> = players
+
+    val players: List<Player>
+        get() = _players.toList()
```

Comment:
이 부분은 내부 리스트가 읽기 전용이기에, 이런 구조를 유지할 필요가 없어보이네요!
```suggestion
    val players: List<Player> = players.toList()
```